src/main/scala/com/databricks/spark/sql/perf/mllib/clustering/LDA.scala

```diff
@@ -0,0 +1,60 @@
+package com.databricks.spark.sql.perf.mllib.clustering
+
+import com.databricks.spark.sql.perf.mllib.{ClassificationContext, TestFromTraining, BenchmarkAlgorithm}
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import org.apache.commons.math3.random.Well19937c
+import org.apache.spark.ml.Transformer
+import org.apache.spark.ml.clustering.{LDA => ML_LDA}
```

Comment:
Because it conflicts with the name of the class below. I am going to remove this import.